plugins/inputs/bond/bond.go

```diff
@@ -245,6 +245,9 @@ func (bond *Bond) gatherSlavePart(bondName string, rawFile string, acc telegraf.
 			fields["failures"] = count
 			if !scanPast {
 				acc.AddFields("bond_slave", fields, tags)
+				fields = map[string]interface{}{
+					"status": 0,
+				}
```

Comment:
Could all three initialization of the `field` variable with a 0 status be moved to the beginning of the for loop to achieve the same thing?

Reply:
i don't think so, since the for loop goes over every line of the bond status output, and each interface might be represented by any number of lines